android/app/src/main/java/app/organicmaps/bookmarks/Holders.java

```diff
@@ -473,6 +472,8 @@ private void bindDescriptionIfEmpty(@NonNull BookmarkCategory category)
 
         Spanned spannedDesc = Utils.fromHtml(desc);
         mDescText.setText(spannedDesc);
+        Linkify.addLinks(mDescText, Pattern.compile("[a-z]+:\\/\\/[^ \\n]*") , "");
```

Comment:
Which other schemes except http(s) can be there? Is file:// scheme supported? What else? 

Why a pattern approach is required?

Reply:
Default linkify doesn't catch URL contains https or https
Linkify doesn't support file scheme -> https://developer.android.com/reference/android/text/util/Linkify and I think it's not possible to copy path of file with default file manager on phone.

Reply:
What about links without http(s) schemes? Will they still be highlighted when the pattern is provided?

Reply:
Yes, you right url without https or https is not catch
this regex is ok? `^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$`

Reply:
It would be great to avoid using regexes completely, they're not great...

Maybe stripping schemes before displaying them helps?

Reply:
We can remove http or https from description content but for example `github.com` is not caught by linkify.
Without my changes, linkify catch only url inside `<a></a>`
Maybe @rtsisyk has an idea to resolve problem?

Reply:
URL without http or https is linkify, but try to open do nothing